website/static/css/site.css

```diff
@@ -2425,3 +2431,9 @@ span#profileFullname{
   padding-left: 20px;
   padding-right: 20px;
 }
+.inline {
+    display: inline-block;
+}
+.hidden-alt {
```

Comment:
Oh, I see: Bootstrap provides `hidden`. Just curious: why can't we use Bootstrap's `hidden` or `invisible`?